Fix name clash with variable PKG_CONFIG_VERSION (used by
FindPkgConfig.cmake to define the minimal required pkg-config version).

This breaks on Gentoo because FindBLAS.cmake uses FindPkgConfig.cmake after
inclusion of this pkg-config.cmake and overwrites PKG_CONFIG_VERSION.

Reported at https://github.com/jrl-umi3218/jrl-cmakemodules/issues/25
This fix is only a workaround.

--- cmake/pkg-config.cmake~	2013-02-08 18:08:24.000000000 +0100
+++ cmake/pkg-config.cmake	2013-09-22 16:21:13.000000000 +0200
@@ -40,7 +40,6 @@
   SET(PKG_CONFIG_PROJECT_NAME "${PROJECT_NAME}")
   SET(PKG_CONFIG_DESCRIPTION "${PROJECT_DESCRIPTION}")
   SET(PKG_CONFIG_URL "${PROJECT_URL}")
-  SET(PKG_CONFIG_VERSION "${PROJECT_VERSION}")
   SET(PKG_CONFIG_REQUIRES "")
   SET(PKG_CONFIG_CONFLICTS "")
   SET(PKG_CONFIG_LIBS "${LIBDIR_KW}${CMAKE_INSTALL_PREFIX}/lib")
--- cmake/pkg-config.pc.cmake~	2013-02-08 18:08:24.000000000 +0100
+++ cmake/pkg-config.pc.cmake	2013-09-22 16:21:25.000000000 +0200
@@ -10,7 +10,7 @@
 Name: ${PKG_CONFIG_PROJECT_NAME}
 Description: ${PKG_CONFIG_DESCRIPTION}
 URL: ${PKG_CONFIG_URL}
-Version: ${PKG_CONFIG_VERSION}
+Version: ${PROJECT_VERSION}
 Requires: ${PKG_CONFIG_REQUIRES}
 Conflicts: ${PKG_CONFIG_CONFLICTS}
 Libs: ${PKG_CONFIG_LIBS}
